src/components/tables/RowActions/DisableEnable/Button.tsx

```diff
@@ -7,7 +7,7 @@ import { specContainsDerivation } from 'utils/misc-utils';
 import { DisableEnableButtonProps } from './types';
 
 const updateShardDisabled = (draftSpec: any, enabling: boolean) => {
-    draftSpec.shards = draftSpec.shards ?? {};
+    draftSpec.shards ??= {};
```

Comment:
What in the magical operator is this? Haha I will definitely be revisiting documentation to see when this was added. This has to mean that `||=`  and `&&=` are valid operators as well... and maybe even oddballs such as `>>=` and `%=`. Wild.

Reply:
Yup... I guess it has been long enough that these can be safely used now. I was holding off for awhile just to be safe but we do really stick to pretty modern browsers so very happy to have these :D .